code/pizzabot.js

```diff
@@ -1,7 +1,62 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
-
-const pizzaPrice = 80
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
 
 //Put your Javscript code here:
+
+const orderPizza = () => {
+  alert(
+    `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`
+  );
+  const orderName = prompt(
+    "Enter the name of the pizza you want to order today."
```

Comment:
it's not so long line, everything can be places in one with ` const orderName = prompt(` and ` );`